graphql-lwt/src/graphql_lwt.ml

```diff
@@ -4,9 +4,10 @@ module Io = Lwt
 
 module Stream = struct
   type 'a io = 'a Lwt.t
-  include Lwt_stream
+  type 'a t = 'a Lwt_stream.t * (unit -> unit)
```

Comment:
Does the "close function" need to be `unit -> unit io`?

Reply:
That may make more sense.